views/events/user_events_telemetry.view.lkml

```diff
@@ -350,6 +350,44 @@ view: user_events_telemetry {
     hidden: no
   }
 
+  dimension: name {
+    label: "Name"
+    description: "The name of the event."
+    type: string
+    sql: ${TABLE}.name;;
+    hidden: no
+  }
+
+  dimension: sign_up_seqence {
+    label: "Chronological Sequence for Cloud Sign-up"
+    type: number
+    sql: CASE WHEN COALESCE(${type}, ${event}) like '%signup%' THEN 1
+      WHEN COALESCE(${type}, ${event}) like '%verify_email%' THEN 2
```

Comment:
Is there a chance a different event in future can contain a `verify_email` string? Similarly for the other strings.

Reply:
For instance, in the future we could be running an A/B test for `pageview_verify_email_a` vs `pageview_verify_email_b`

Reply:
yes, totally possible, in which case we would update the mapping here accordingly.

Reply:
What are your thoughts on using the exact string values instead of using the `WHERE ... LIKE` expression?

Reply:
I used the exact string values initially, but later updated as I'm not close to sensitive event changes, at the moment.

Reply:
I can make the update to the exact string, if you'd like.

Reply:
> but later updated as I'm not close to sensitive event changes, at the moment.

Does this mean we don't anticipate the event strings to change? 

While that may be true, it'll still be less error-prone if in the future we add a new one with a similar name but forget to add it to the chart,

Reply:
So I'm leaning to using the exact string, unless you see disadvantages with it?

Reply:
The logic is updated.